receiver/jaegerreceiver/factory.go

```diff
@@ -128,11 +128,23 @@ func createDefaultConfig() configmodels.Receiver {
 			ThriftHTTP: &confighttp.HTTPServerSettings{
 				Endpoint: defaultHTTPBindEndpoint,
 			},
-			ThriftBinary: &confignet.TCPAddr{
+			ThriftBinary: &ProtocolUDP{
 				Endpoint: defaultThriftBinaryBindEndpoint,
+				ServerConfigUDP: ServerConfigUDP{
```

Comment:
Since this is the same for compact and thrift it might be good to have it in a common factory functon.

Reply:
Yep, seems good